cmd/buck/cli/cli.go

```diff
@@ -97,3 +100,19 @@ func printLinks(reply *pb.LinksReply) {
 		cmd.Message("%s Bucket website", aurora.White(reply.WWW).Bold())
 	}
 }
+
+func setCidVersion(buck *local.Bucket, key string) {
```

Comment:
This does nothing if the local bucket root is defined. Otherwise, it will get the root and cid type from remote.